### PR TITLE
Added files and setup pages for 1.19 commands and GriefDefender

### PR DESCRIPTION
**Pull request by Hanoverfist**
Pages are marked as WIP

### CODE REVIEW

File: docs/source/Knowledge_Base/commands_1.19.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
++++++++++++++
```
Comment (Aiidoneus) [inline docs/source/Knowledge_Base/commands_1.19.rst:1]: Format Error

File: docs/source/Knowledge_Base/commands_1.19.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Commands 1.19 [Work in Progress]
++++++++++++++
```
Comment (Aiidoneus) [inline docs/source/Knowledge_Base/commands_1.19.rst:3]: Format Error

File: docs/source/Knowledge_Base/griefdefender.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.. _ref-griefdefender:
+
++++++++++++++++++++++++++++++++++++
```
Comment (Aiidoneus) [inline docs/source/Knowledge_Base/griefdefender.rst:3]: Format issue

File: docs/source/Knowledge_Base/griefdefender.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+GriefDefender (protect your land) [Work in Progress]
++++++++++++++++++++++++++++++++++++
```
Comment (Aiidoneus) [inline docs/source/Knowledge_Base/griefdefender.rst:5]: Format Issue

**Review by Aiidoneus — CHANGES_REQUESTED** [review]
These need to be fixed

File: docs/source/Knowledge_Base/commands_1.19.rst (hunk 1 above)
Comment (Hanoverfist) [reply docs/source/Knowledge_Base/commands_1.19.rst]: Fixed in both files.

File: docs/source/Knowledge_Base/commands_1.19.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* ``/gd claim farewell [message]`` - Changes the leave message of your claim
+* ``/gd claim greeting [message]`` - Changes the enter message of your claim
+
+See -placeholder for GD link- for more information.
+
+Tickets
+=======
+* ``/ticket`` - Shows information regarding ticket creation
+
+See :ref:`create-ticket` for more information.
```
[the remark this answers is not in the document]
Comment (Hanoverfist) [reply docs/source/Knowledge_Base/commands_1.19.rst]: Removed

File: docs/source/Knowledge_Base/griefdefender.rst (hunk 3 above)
Comment (Hanoverfist) [reply docs/source/Knowledge_Base/griefdefender.rst]: Fixed

**Comment by Hanoverfist** [comment]
wont let me make any requests..

**Pull request closed without merging**